scalameta/scalameta/shared/src/test/scala/scala/meta/tests/prettyprinters/SyntacticSuite.scala

```diff
@@ -436,7 +436,7 @@ class SyntacticSuite extends scala.meta.tests.parsers.ParseSuite {
   test("package foo; class C; package baz { class D }") {
     val tree = source("package foo; class C; package baz { class D }")
     assert(tree.structure === "Source(List(Pkg(Term.Name(\"foo\"), List(Defn.Class(Nil, Type.Name(\"C\"), Nil, Ctor.Primary(Nil, Name(\"\"), Nil), Template(Nil, Nil, Self(Name(\"\"), None), Nil)), Pkg(Term.Name(\"baz\"), List(Defn.Class(Nil, Type.Name(\"D\"), Nil, Ctor.Primary(Nil, Name(\"\"), Nil), Template(Nil, Nil, Self(Name(\"\"), None), Nil))))))))")
-    assert(tree.syntax === "package foo\nclass C\npackage baz {\n  class D\n}")
+    assert(tree.syntax === s"package foo${EOL}class C${EOL}package baz {$EOL  class D${EOL}}")
```

Comment:
I find it a bit unsettling that `tree.syntax` is OS dependent, would it make sense if TreeSyntax emitted \n instead of EOL?

Reply:
I know so little about the actual library that I don't really have an opinion on that in this specific case.

In the general case (perhaps surprisingly), things tend to get more rather then less portable if `\n` is consistently used for everything, as long as notepad isn't the tool of choice of the Windows user (and it never is).

Unfortunately things like `println` are platform specific, making you in some cases end up with mixed line endings if you attempt to do that.

I could experiment with globally replacing EOL with \n and see where that brings us - maybe it removes the need for this PR almost entirely.

One notable issue is git. In some configurations, git on windows will replace \n line endings with \r\n. This is problematic for scala in general, since line endings may in some cases (for example in multiline strings) not be interchangeable.

Reply:
Thanks for sharing your view. I propose we don't change the pretty-printing in this PR and focus on getting tests passing in Windows, as you've done. Globally EOL with \n  might be worth it IMO, I'm happy to explore that path separately later.